composer.json

```diff
@@ -73,7 +73,7 @@
         "sylius-labs/doctrine-migrations-extra-bundle": "^0.1.4 || ^0.2",
         "sylius-labs/polyfill-symfony-event-dispatcher": "^1.0.1",
         "sylius-labs/polyfill-symfony-framework-bundle": "^1.0 || ^1.1",
-        "sylius-labs/polyfill-symfony-security": "^1.1",
+        "sylius-labs/polyfill-symfony-security": "dev-master",
```

Comment:
You can revert this change as the new version of polyfill has been released: https://github.com/SyliusLabs/PolyfillSymfonySecurity/releases/tag/v1.1.2